specification/consumption/resource-manager/Microsoft.Consumption/stable/2018-01-31/consumption.json

```diff
@@ -117,6 +117,79 @@
         }
       }
     },
+    "/{scope}/providers/Microsoft.Consumption/marketplaces": {
+      "get": {
+        "tags": [
+          "Marketplaces"
+        ],
+        "operationId": "Marketplaces_List",
```

Comment:
[Odata extension](https://github.com/Azure/autorest/blob/master/docs/extensions/readme.md#x-ms-odata) could be useful here :D

Reply:
@mcardosos done.